crates/proof/executor/src/syscalls/tx_env.rs

```diff
@@ -41,7 +41,7 @@ pub(crate) fn fill_tx_env_for_contract_call(
         blob_hashes: Vec::new(),
         max_fee_per_blob_gas: None,
         optimism: OptimismFields {
-            source_hash: None,
+            source_hash: Some(B256::ZERO),
```

Comment:
Syscalls are free; The operator fee is always charged to non-deposits, and can have a non-zero constant floor.

Reply:
We really should put this in a comment for posterity but can be a follow-up pr